lib/modules/apostrophe-ui/public/css/components/tooltip.less

```diff
@@ -0,0 +1,35 @@
+.apos-tooltip
```

Comment:
Using spaces, tab size 2. I have no idea what the house style is though. Is that what your asking?